### PR TITLE
Add Intro details

### LINKED ISSUE #96: Intro Details, Dedication, Copyright Notice, etc.
There's some info on the [first two pages](https://freedom.press/sites/default/files/encryption_works.pdf) of the existing Encryption Works PDF that we should loop into the new version:

- Title (which we've agreed on)
- August 2015.
- (Next page) Dedication. I'd like something like "Dedicated to cypherpunks, and to whistleblowers past and future," but comments welcome. 
- Authorship Details -- I need to talk to Trevor about this, but I'm guessing it'll be something like "Written by Micah Lee, edited by Tommy Collison, Harlo Holmes, and Garrett Robinson"?
- "A publication of **Freedom of the Press Foundation**. Originally published July 2013, this version August 2015," or some such.
- Copyright notice probably fine as-is.
- Info on contributing to the github.

### PR DESCRIPTION
**Pull request by tommycollison**
Add details such as the updated date, editorship deets, changing the dedication, and adding back the copyright notice and a call for GitHub contributors. Resolves #96.

### CODE REVIEW

**Comment by garrettr** [comment]
@tommycollison You need to rebase this branch so it applies cleanly to the current master branch.

**Comment by tommycollison** [comment]
@garrettr Done.

File: encryption_works.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+div#out-of-date {
+  background-color: #FFF263;
+  border: 1px solid #EFC933;
+  padding: 0px 16px;
+}
+</style>
+
+<div id="out-of-date">
+<p>Warning: This guide has not updated in over a year. Freedom of the Press Foundation is working on an updated version. If you're interested in contributing, or have ideas for what this guide should cover, please submit issues on <a href="https://github.com/freedomofpress/encryption-works">GitHub</a>
+</div>
```
Comment (garrettr) [inline encryption_works.md]: This PR re-adds the "out-of-date" warning that was just removed in #90. This looks like an incorrect resolution of the merge conflict you were having earlier.

**Comment by tommycollison** [comment]
Ah, I need to roll back, evidently didn't `git pull`.

**Comment by harlo** [comment]
point people to our contributing file instead...

**Comment by tommycollison** [comment]
Done!

**Pull request merged**